src/components/addSourceWizard/addSourceWizardStepOne.js

```diff
@@ -25,7 +25,12 @@ class AddSourceWizardStepOne extends React.Component {
       <FormState validateOnMount setValues={{ sourceType: type }} validate={this.isStepValid}>
         {({ values, handleOnEvent, handleOnSubmit }) => (
           <Form isHorizontal onSubmit={handleOnSubmit}>
-            <FormGroup role="radiogroup" isStack label={t('form-dialog.label', { context: 'source-type' })}>
+            <FormGroup
+              role="radiogroup"
+              fieldId="sourceType"
+              isStack
+              label={t('form-dialog.label', { context: 'source-type' })}
+            >
```

Comment:
It appears there may be a misunderstanding of the FormGroup components `fieldId` behavior. The current behavior was completed under #217. 

> The FormGroup component was designed to automagically apply the First Childs id consistently so a developer did not have to maintain 2 separate identifiers, which when dealing with form fields can become tedious and prone to errors.

There appears to be an unclear reason associated with this update. Why does this the `fieldId` for `addSourceWizardStepOne.js` have to change when there is already a generated ID that appears to be consistent?

Reply:
I was looking at [PatternFly docs](http://v4-archive.patternfly.org/v4/components/form/#formgroup), which claim that `fieldId` is "required when a role of "group" or "radiogroup" is passed in." That's why added it, as I noticed it's not specified and generated one is the same as value of first item (which is strange, if harmless).

Now I see that most / all components here are custom components that are thin wrappers around PatternFly components. So we can skip `fieldId`, but one will be generated before we use PatternFly component.

Removed in ae106e3f01300fe89c58b0c8cf1ef74a584fb4cc . I'm adding commits to make review easier, they should be all squashed before / during merge.